packages/compass-import-export/src/modules/export.js

```diff
@@ -126,12 +128,18 @@ const reducer = (state = INITIAL_STATE, action) => {
   }
 
   if (action.type === ON_MODAL_OPEN) {
-    return {
+    const newState = {
       ...INITIAL_STATE,
       count: action.count,
-      query: action.query,
+      query: action.query || null,
+      aggregation: action.aggregation || null,
       isOpen: true,
     };
+
+    if (action.aggregation) {
+      newState.exportStep = EXPORT_STEP.FILETYPE;
+    }
```

Comment:
I think the plan was to still have one screen that will show the aggregation that will be exported and this doesn't look like it's happening right now: we skip right to file type selection. Was there a decision to change the plan or are we planning to do this in a separate PR?

Reply:
Thanks for pointing that out. I created a follow up: [COMPASS-5735](https://jira.mongodb.org/browse/COMPASS-5735)